gapic/src/main/com/google/gapid/perfetto/Unit.java

```diff
@@ -195,13 +195,18 @@ public String format(double value) {
     }
 
     public static class Percent extends Simple {
+      private static final Double CUT_OFF = 0.05;
       public Percent() {
         super("%");
       }
 
       @Override
       public String format(double value) {
-        return (value == 100) ? super.format(100) : super.format(value);
+        if (value == 100) {
+          return super.format(100);
+        } else {
+          return Math.abs(value) < CUT_OFF ? String.format("%,.2g%%", value) : String.format("%,.2f%%", value);
```

Comment:
You don't need to do the abs here, as it's done above - the formatter should only be getting positive values.
Also, the format should be `"%,g%%"`.

Reply:
Thanks for being patient here! Just done it! 👍